modules/ROOT/pages/scripting/scripting-module.adoc

```diff
@@ -2,20 +2,98 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: Scripting, custom code, groovy, ruby, phyton, javascript, java
+:keywords: Scripting, custom code, groovy, ruby, phyton, javascript
 
-////
-*Reference:* xref:vm/vm-reference.adoc[VM Connector Reference]
-////
-
-The scripting module can execute custom logic written in a scripting language. Supported scripting languages:
+In spite of the rich selection of elements available in Mule, you may find yourself in need of creating custom code to carry out all or part of a complex task, or in case you want to reuse some modules that you have already writen. The scripting module can execute custom logic written in a scripting language. Supported scripting languages:
 
 * Groovy
 * Python
 * Javascript
 * Ruby
 
-//TODO take use cases from mule 3 docs
+== Instalation
+
+The scripting module is already included in Anypoint Studio. In case you want to add the component into one of your flows, just search for it in the Mule Palette.
+
+image::scripting/scripting-flow.png[Scripting Execute component]
+
+== Basic Usage
+
+To configure the Script component, double-click the Script Execute element to open the **Mule Properties Editor**. Use the **General** tab to specify your code on the script text window.
+
+Choose the Engine for the execute component:
+
+* Groovy
+* Nashorn (JavaScript)
+* Jython (Python)
+* JRuby (Ruby)
+
+You should enter de scripting code. In case you want to load the code from an external file, you can set this by using a xref:4.1@mule-runtime::configuring-properties.adoc[File Configuration Property].
```

Comment:
```suggestion
You should enter the scripting code. In case you want to load the code from an external file, you can set this by using a xref:4.1@mule-runtime::configuring-properties.adoc[File Configuration Property].
```